RELEASE.md

```diff
@@ -0,0 +1,100 @@
+# sBTC: Release Process
+
+## Executive summary
+
+This release process targets the following goals:
+
+1. Ensure there exists a provable chain of trust connecting the source code to
+   the corresponding artifacts.
+1. Ensure there exists a clear _separation of duties_ between those who _write_
+   the code and those who _release_ it (and announce its release).
+
+It implements (1) by leveraging GitHub rulesets for branch/tag protection and
+attestation for artifacts and (2) through the announcement process (described
+below).
+
+## [sBTC Core developers] Creating a new release
+
+An sBTC release is a tagged commit on the `main` branch.
+
+Any commit to `main` SHALL require at least one review through a GitHub pull
```

Comment:
I'd change the `SHALL`s to `MUST`s:

> "Must" is used to denote a mandatory term. However, “must” indicates that a requirement is non-negotiable, unequivocal, and imperative. Although both “must” and “shall” describe required obligations. Most interpret “must” to convey more urgency and importance than "shall."

Or if you prefer `SHALL` then I would include [clear definitions](https://datatracker.ietf.org/doc/html/rfc2119) in the prelude.

Reply:
Done!